monkestation/code/modules/liquids/code/liquid_systems/liquid_controller.dm

```diff
@@ -0,0 +1,114 @@
+SUBSYSTEM_DEF(liquids)
```

Comment:
One concern of mine is how much processing this subsystem will actually take.
Bee is not in a great place, performance-wise.

The speed that it updates will likely need to be tweaked so that it doesn't take up too much processing, even if the liquids end up being weirdly slow.

Reply:
yea i only tested this on a self host and it was fine but i can see it maybe causing issues on high pop i need to do a test with more than just me to actually see that though